esx_service/vmdk_ops.py

```diff
@@ -792,20 +822,34 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
         vol_name, datastore = parse_vol_name(full_vol_name)
     except ValidationError as ex:
         return err(str(ex))
-    if not datastore:
-        datastore = default_datastore
 
-    if not vmdk_utils.validate_datastore(datastore):
+    if datastore and not vmdk_utils.validate_datastore(datastore):
```

Comment:
Maybe it's better to hide a check for `if datastore` inside of validate datastore, which should always return False if passed None ? (depends on other places in the code, so it's a nit)